docs/guides/authoring/overview.rst

```diff
@@ -0,0 +1,328 @@
+..
+  SPDX-License-Identifier: CC-BY-4.0
+  Copyright Contributors to the OpenColorIO Project.
+
+.. _config_overview:
+
+
+
+Config syntax
+=============
+
+OpenColorIO is primarily controlled by a central configuration file,
+usually named ``config.ocio``. This page will only describe how to
+*syntactically* write this OCIO config - e.g. what transforms are
+available, or what sections are optional.
+
+This page alone will not help you to write a useful config file! See
+the :ref:`configurations` section for examples of complete, practical
+configs, and discussion of how they fit within a facilities workflow.
+
+YAML basics
+***********
+
+This config file is a YAML document, so it is important to have some
+basic knowledge of this format.
+
+The `Wikipedia article on YAML <http://en.wikipedia.org/wiki/YAML>`__
+has a good overview.
+
+OCIO configs typically use a small subset of YAML, so looking at
+existing configs is probably the quickest way to familiarise yourself
+(just remember the indentation is important!)
+
+Checking for errors
+*******************
+
+Use the ``ociocheck`` command line tool to validate your config. It
+will inform you of YAML-syntax errors, but more importantly it
+performs various OCIO-specific "sanity checks".
+
+For more information, see the overview of :ref:`overview-ociocheck`
```

Comment:
Worth mentioning the use of 
```
:py:meth:`Config.validate`
```
here in a future revision. Since _most_ config authoring is done via the Python API, this is often more convenient that ``ociocheck`` in practice. Good to mention both though for sure.

Reply:
Though now that I think of it, ``ociocheck`` does quite a bit more than ``Config.validate`` (formerly ``Config.sanityCheck``). May still be worth mentioning in the future though.